app/webpacker/lib/requests/routes.js.erb

```diff
@@ -195,6 +195,7 @@ export const apiV0Urls = {
   },
   delegates: {
     list: `<%= CGI.unescape(Rails.application.routes.url_helpers.api_v0_delegates_path)%>`,
```

Comment:
Shouldn't `delegates.list` be removed if they are no longer used?

Reply:
I can't tell with certainty whether it's no longer used. And I don't want to bother checking all other JS webpacker components.